app/content/carto-user-manual/maps/publishing-and-sharing-maps.md

```diff
@@ -35,6 +36,14 @@ Public maps have some extra options: to share the link with the public, to get D
 
 ![Publish options share link](/img/cloud-native-workspace/maps/map_new_sharing_options_developers.png)
 
+### Sharing with certain groups
+
+If you have Single-Sign On (SSO) enabled at your organization, you will be able to share maps only with specific groups inside your organization, instead with all users belonging to it.
+
+![Map sharing with specific groups](/img/cloud-native-workspace/maps/map_sharing_options_groups.png)
```

Comment:
I updated the img, because it is number of groups instead of number of users. You can find the img in the same path, with this name: map_sharing_options_with_groups.png

Reply:
> I updated the img

I can't see it in a commit, did you update the image in the repo?